src/fhir/medications.test.ts

```diff
@@ -0,0 +1,71 @@
+import { MedicationStatement } from "fhir/r4";
+import { splitSummarizedMedications } from "./medications";
+import { SYSTEM_RXNORM } from "./system-urls";
+
+describe("splitSummarizedMedications", () => {
+  test("splits medications correctly", () => {
+    const summarizedMeds: MedicationStatement[] = [
+      {
+        resourceType: "MedicationStatement",
+        status: "active",
+        subject: {
+          reference: "Test",
+        },
+        medicationCodeableConcept: {
+          coding: [
+            {
+              system: SYSTEM_RXNORM,
+              code: "known",
+            },
+          ],
+        },
+      },
+      {
+        resourceType: "MedicationStatement",
+        status: "active",
+        subject: {
+          reference: "Test",
+        },
+        medicationCodeableConcept: {
+          coding: [
+            {
+              system: SYSTEM_RXNORM,
+              code: "unknown",
+            },
+          ],
+        },
+      },
+    ];
+
+    const builderMeds: MedicationStatement[] = [
+      {
+        resourceType: "MedicationStatement",
+        status: "active",
+        subject: {
+          reference: "Test",
+        },
+        medicationCodeableConcept: {
+          coding: [
+            {
+              system: SYSTEM_RXNORM,
+              code: "known",
+            },
+          ],
+        },
+      },
+    ];
+
+    const { builderMedications, otherProviderMedications } =
+      splitSummarizedMedications(summarizedMeds, builderMeds);
+
+    expect(builderMedications.length).toBe(1);
+    expect(otherProviderMedications.length).toBe(1);
```

Comment:
just so failures don't read "expected undefined to be 1" or "expected "2 to be 1"
```suggestion
    expect(builderMedications).toHaveLength(1);
    expect(otherProviderMedications).toHaveLength(1);
```